fastly/block_fastly_service_v1_director_test.go

```diff
@@ -215,7 +215,7 @@ func TestAccFastlyServiceV1_directors_basic(t *testing.T) {
 	})
 }
 
-func testAccCheckFastlyServiceV1DirectorsAttributes(service *gofastly.ServiceDetail, directors []*gofastly.Director, director_backends []*gofastly.DirectorBackend) resource.TestCheckFunc {
+func testAccCheckFastlyServiceV1DirectorsAttributes(service *gofastly.ServiceDetail, directors []*gofastly.Director, _ []*gofastly.DirectorBackend) resource.TestCheckFunc {
```

Comment:
Should we not just update the caller of this function to not pass the last argument considering it's not ever used? 

e.g. the `_ []*gofastly.DirectorBackend`

Reply:
This makes sense and it has been removed.  I have pushed after confirming the test passed.